lib/web_ui/lib/src/engine/layers.dart

```diff
@@ -414,6 +446,10 @@ abstract class LayerOperation {
   void post(SceneCanvas canvas, ui.Rect contentRect);
 
   PlatformViewStyling createPlatformViewStyling();
+
+  // Most operations don't need to actually be performed at all if they don't
+  // contain any pictures.
```

Comment:
Consider making it a full `///` dart doc, and also include the explanation of the effect this field has (the context is nice though; let's keep it).